ccloud/docs/includes/set-acls-origin-and-destination.rst

```diff
@@ -0,0 +1,35 @@
+Since the origin cluster and destination cluster in this example are both |ccloud|, configure appropriate ACLs for the service account ids corresponding to |crep| in both |ccloud| clusters.
+For more details on |crep| ACLs, see :ref:`replicator_security_overview`.
+
+In the origin |ccloud|:
+
+.. sourcecode:: bash
+
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation CREATE --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation WRITE --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation READ --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE-CONFIGS --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation ALTER-CONFIGS --topic <topic-origin>
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE --cluster-scope
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation CREATE --cluster-scope
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation CREATE --topic __consumer_timestamps
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation WRITE --topic __consumer_timestamps
+   ccloud kafka acl create --allow --service-account <service-account-id-origin> --operation DESCRIBE --topic __consumer_timestamps
+
+
+In the destination |ccloud|:
+
+.. sourcecode:: bash
+
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation CREATE --topic <topic-destination>
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation WRITE --topic <topic-destination>
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation READ --topic <topic-destination>
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation DESCRIBE --topic <topic-destination>
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation DESCRIBE-CONFIGS --topic <topic-destination>
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation ALTER-CONFIGS --topic <topic-destination>
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation DESCRIBE --cluster-scope
+   ccloud kafka acl create --allow --service-account <service-account-id-destination> --operation CREATE --cluster-scope
```

Comment:
Again as stated before, if this is specifically for Replicator, giving `CREATE` on the topic should be sufficient.